custom/bin/ARE/web/webapps/asterics-smarthome-knx/index.html

```diff
@@ -26,6 +26,53 @@
 		<!-- Modal styleheet -->
 		<link rel="stylesheet" href="../startpage/style/modal.css">
 
+		<!-- Bootstrap core JavaScript
+		================================================== -->
+		<!-- jQuery first, then Popper.js, then Bootstrap JS -->
+			<!-- <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script> -->
+		<!-- <script src="../startpage/lib/jquery-3.2.1.slim.min.js"></script> -->
+		<script src="../startpage/lib/jquery-3.2.1.js"></script>
+			<!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script> -->
+		<script src="../startpage/lib/popper.min.js"></script>
+			<!-- <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script> -->
+		<script src="../startpage/lib/bootstrap.min.js"></script>
+		
+		<!-- Modal image
+		================================================== -->
+		<!-- Modal image container -->
+		<div id="modalwrapper" class="modal">
+				<span class="close" onclick="document.getElementById('modalwrapper').style.display='none'">&times;</span>
+				<img class="modal-content" id="modalimage">
+				<div id="caption"></div>
+		</div>
+		<!-- Modal image action script -->
+		<script src="../startpage/lib/modal.js"></script>
+		<!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script> -->
+	
+	
+		<!-- AsTeRICS
+		================================================== -->
+		<!-- provided from AsTeRICS 3.0 -->
+		<!-- jquery is already loaded above -->
+		<!-- <script type="text/javascript" src="../startpage/lib/jquery-3.2.1.min.js"></script> -->
+		<script src="../startpage/clientExample/javascript/JSmap.js"></script>
+		<script src="../startpage/clientExample/javascript/areCommunicator.js"></script>
+	
+		<!-- provided by this repository, should be part of the framework later -->
+		<script src="../startpage/lib/webAppUtils.js"></script>
+		<script src="../startpage/lib/modelManipulation.js"></script>
+		
+		<script type="text/javascript">
+			//Set the base URI of the running ARE instance.     
+			//You could also make this configurable by the user.
+			setBaseURI("http://localhost:8081/rest/");
+	
+			//Define file to path model file on the webserver.
+			var modelFilePathOnWebserver='webapps/asterics-solution-template/models/nameOfModelFile.acs';
```

Comment:
You must provide your model file in 
```webapps/asterics-smarthome-knx/models/<xxx>.acs```

and set the path of variable ```modelFilePathOnWebserver``` accordingly.

Maybe you forget to commit and push something???